src/tools/rest-frontend/resources/assets/js/controllers/main/MainController.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = function ($rootScope, $scope, Logger, $state, $anchorScroll, webStructure) {
+
+    var vm = this;
+    $scope.$rootScope = $rootScope;
+
+    // build the dynamic menu
+    $scope.menu = webStructure;
+
+//        vm.currentLanguage = 'de';
```

Comment:
It's actually c&p from another project from me, as preparation for a change-language function that can be added. Default language and such stuff is all defined in the configuration file.